forge-gui/src/main/java/forge/gamemodes/match/HostedMatch.java

```diff
@@ -155,6 +155,7 @@ public void startGame() {
 
         game = match.createGame();
         game.EXPERIMENTAL_RESTORE_SNAPSHOT = FModel.getPreferences().getPrefBoolean(FPref.MATCH_EXPERIMENTAL_RESTORE);
+        game.AI_TIMEOUT = FModel.getPreferences().getPrefInt(FPref.MATCH_AI_TIMEOUT);
```

Comment:
this logic doesn't work because the players get created with the default before
(It's probably a good idea to have more than one value though in case we want to combine it with AI difficulty or something later)

Reply:
remove the definition on player and just take the value from the game